packages/components/Button/src/Button.settings.android.ts

```diff
@@ -0,0 +1,88 @@
+import { buttonName, IButtonType } from './Button.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+
+export const settings: IComposeSettings<IButtonType> = [
+  {
+    tokens: {
+      backgroundColor: 'buttonBackground',
+      color: 'buttonText',
+      borderColor: 'buttonBorder',
+      borderWidth: 1,
+      borderRadius: 12,
+    },
```

Comment:
I still hold that adding overrides here in this platform specific file is mixing info that should be coming from a theme into the base, theme-less component. When I spoke with Tushar offline about this, I think a medium compromise we talked about was that we need more control token mappings in the theme so that we can just set the tokens in the theme rather than override them the `components` section of a theme's definition (which increases bundle size). 

For this specific PR, I'd rather we add more control tokens for 'buttonBorderRadius` or `buttonBorderWidth` in `@fluentui-react-native/theme-types` than introduce them here

Reply:
>I still hold that adding overrides here in this platform specific file is mixing info that should be coming from a theme into the base, theme-less component.

I am planning to do this in a change to move the current theme to v-next (probably identifying this as the first step for token pipeline integration). The end goal is to fetch the information from the theme as mentioned.